client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -115,21 +116,23 @@ const EmailProvidersStackedComparison: FunctionComponent< EmailProvidersStackedC
 				{ translate( 'Pick an email solution' ) }
 			</h1>
 
-			<div className="email-providers-stacked-comparison__how-they-compare">
-				{ translate( 'Not sure how to start? {{a}}See how they compare{{/a}}.', {
-					components: {
-						a: (
-							<a
-								href={ emailManagementInDepthComparison(
-									siteName,
-									selectedDomainName,
-									currentRoute
-								) }
-							/>
-						),
-					},
-				} ) }
-			</div>
+			{ isEnabled( 'emails/in-depth-comparison' ) && (
+				<div className="email-providers-stacked-comparison__sub-header">
+					{ translate( 'Not sure how to start? {{a}}See how they compare{{/a}}.', {
```

Comment:
Nit: the original design has the wording as "Not sure where to start?". It's probably worth correcting that sooner rather than later.

Also looking at the design, I think this text should be a bit smaller and should have more vertical whitespace above and below it. But none of this is blocking -- these were existing issues. Design: pcDL0W-Kp-p2

Reply:
Good catch!